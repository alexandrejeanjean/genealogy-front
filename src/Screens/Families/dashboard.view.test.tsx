import React from "react";
import { shallow } from "enzyme";
import DashboardView from "./dashboard.view";

const list = [{ id: 0, name: "FamilyTest" }];

const DashboardViewProps = {
  setToastVisible: jest.fn(),
  list: list,
  goTo: jest.fn(),
  deleteFamily: jest.fn(),
  setModalShow: jest.fn(),
  modalShow: false,
  createFamily: jest.fn(),
};

describe("DashboardView", () => {
  it("should render correctly", () => {
    const Component = shallow(<DashboardView {...DashboardViewProps} />);
    expect(Component).toMatchSnapshot();
  });
});
