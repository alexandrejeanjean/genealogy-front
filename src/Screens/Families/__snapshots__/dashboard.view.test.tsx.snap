// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardView should render correctly 1`] = `
<Fragment>
  <section>
    <ListWrapper
      datas={
        Array [
          Object {
            "id": 0,
            "name": "FamilyTest",
          },
        ]
      }
      getAvatar={[Function]}
      handleClick={[Function]}
      handleDelete={[Function]}
    >
      <CreateItemBtn
        handleClick={[Function]}
      />
    </ListWrapper>
  </section>
  <ModalForm
    inputs={
      Array [
        Object {
          "inputType": "text",
          "name": "name",
          "placeholder": "Ex: Armstrong",
        },
      ]
    }
    onHide={[Function]}
    show={false}
    submit={[MockFunction]}
    title="Family"
  />
</Fragment>
`;
